### PR TITLE
Added an article How to Customize Thank You page.(#36)

### PR DESCRIPTION
**Pull request by shreeshjha**
I was able to create an issue and also add up the article about "**How to customize the thank you page**"
**Here is the link for the issue I created**:- (#36)

### CODE REVIEW

File: event-setup/Customizing -ThankYou-Page.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Customization lets the developer construct their own personal choices about what they want to see, or set preferences for how preferred information is organized or shown. It enhances users experience because it allows users to control their interaction.
+
+Thank You Page Customizing the Thank You Page of the Fossasia Page doesn’t requires any high knowledge of JavaScript and Css it require some translation tools so it can translate the thank you statement in different language. The translation can be added in Github Repository :- https://github.com/fossasia/gci18.fossasia.org After this the only thing left is to make changes in the thankyou.js file. Follow the steps shown below in the pictures and follow the pattern of entering the translation in particular manner as shown in thankyou.js file.
```
Comment (jogendra) [inline event-setup/Customizing -ThankYou-Page.md]: Change `JavaScript` to `Javascript`

Comment (jogendra) [inline event-setup/Customizing -ThankYou-Page.md]: Change `Css` to `CSS`

**Comment by shreeshjha** [comment]
I've done the following changes as mentioned please go through it and inform me whether more updates are required or not

File: event-setup/Customizing -Thank-You-Page.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+Customization lets the developer construct their own personal choices about what they want to see, or set preferences for how preferred information is organized or shown. It enhances users experience because it allows users to control their interaction.
+
+Thank You Page Customizing the Thank You Page of the Fossasia Page doesn’t requires any high knowledge of Javascript and CSS it require some translation tools so it can translate the thank you statement in different language. The translation can be added in Github Repository :- https://github.com/fossasia/gci18.fossasia.org After this the only thing left is to make changes in the thankyou.js file. Follow the steps shown below in the pictures and follow the pattern of entering the translation in particular manner as shown in thankyou.js file.
```
[the remark this answers is not in the document]
Comment (mayank8318) [reply event-setup/Customizing -Thank-You-Page.md]: @Masquerade0097 agreed :)

[the remark this answers is not in the document]
Comment (shreeshjha) [reply event-setup/Customizing -Thank-You-Page.md]: I have removed the Thank You page which was not appropriate for there.

File: event-setup/Customizing -Thank-You-Page.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+Customization lets the developer construct their own personal choices about what they want to see, or set preferences for how preferred information is organized or shown. It enhances users experience because it allows users to control their interaction.
+
+Thank You Page of the Fossasia Page doesn’t requires any high knowledge of Javascript and CSS it require some translation tools so it can translate the thank you statement in different language. The translation can be added in Github Repository :- https://github.com/fossasia/gci18.fossasia.org After this the only thing left is to make changes in the thankyou.js file. Follow the steps shown below in the pictures and follow the pattern of entering the translation in particular manner as shown in thankyou.js file.
```
[the remark this answers is not in the document]
Comment (shreeshjha) [reply event-setup/Customizing -Thank-You-Page.md]: Done :)
Anything else

**Pull request merged**